### PR TITLE
Overlay gather

### PR DESCRIPTION
**Pull request by dsusviela**
Modified:
  - Gather.js
    - Added GatherOverlay component
    - Changed class name from 'App' to 'Gather'
  - Gather/styles.js
    - Added several styles for the overlay
  - localization/en.js and   - localization/es.js:
    - Fixed several errors with eslist and prettier.
    - Added strings for month names
    - Added strings for overlay button

New: 
  - GatherOverlay.js

### CODE REVIEW

File: src/components/Gather/Gather.js [hunk 1]
```diff
@@ -16,10 +16,12 @@ import Logo01 from '../../assets/images/Logo01.png';
 import user128 from '../../assets/ic_user/ic_user128.png';
 import sideMenuIcon from '../../assets/ic_common/ic_hamburger.png';
 import stylesGather from './styles';
+import GatherOverlay from './GatherOverlay';
+
 
 Mapbox.setAccessToken('pk.eyJ1IjoicXFtZWxvIiwiYSI6ImNqbWlhOXh2eDAwMHMzcm1tNW1veDNmODYifQ.vOmFAXiikWFJKh3DpmsPDA');
 
-class Gather extends Component {
+export default class Gather extends Component {
```
Comment (anitanigma) [inline src/components/Gather/Gather.js]: Gather is exported in line 166

File: src/localization/es.js [hunk 2]
```diff
@@ -20,6 +20,26 @@ export default {
...
   baleTitle: 'Enfardar',
+  month0: 'Enero',
+  month1: 'Febrero',
+  month2: 'Marzo',
+  month3: 'Abril',
+  month4: 'Mayo',
+  month5: 'Junio',
+  month6: 'Julio',
+  month7: 'Agosto',
+  month8: 'Setiemre',
```
Comment (anitanigma) [inline src/localization/es.js]: month8: 'Setiembre'

File: src/localization/es.js [hunk 3]
```diff
@@ -20,6 +20,26 @@ export default {
...
+  month9: 'Octubre',
+  month10: 'Noviembre',
+  month11: 'Diciembre',
+  day0: 'Domingo',
+  day1: 'Lunes',
+  day2: 'Martes',
+  day3: 'Miercoles',
```
Comment (anitanigma) [inline src/localization/es.js]: day3: Miércoles

**Comment by dsusviela** [comment]
There's a known issue with StackNavigator and pushing several times the Gather screen. The issue will not be resolved on this branch, but in another one, and then it will be merged here.

File: src/components/Gather/GatherOverlay.js [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+import stylesGather from './styles';
+
+const transformTime = (min) => {
+  if ((min / 10) >= 1) {
+    return (min.toLocaleString());
+  }
+  return (`0${min}`);
+};
+
+const transformDay = (day) => {
```
Comment (anitanigma) [inline src/components/Gather/GatherOverlay.js]: Days are one day ahead - Sunday shows Monday, etc

**Pull request merged**